MapboxNavigation/RouteVoiceController.swift

```diff
@@ -45,15 +51,27 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     public var instructionVoiceVolume = "x-loud"
     
     
-    public init(regionType: AWSRegionType = AWSRegionType.USEast1, identityPoolId: String?) {
-        if let identityPoolId = identityPoolId {
-            self.useDefaultVoice = false
-            let credentialsProvider = AWSCognitoCredentialsProvider(regionType:regionType, identityPoolId: identityPoolId)
-            let configuration = AWSServiceConfiguration(region:regionType, credentialsProvider:credentialsProvider)
-            AWSServiceManager.default().defaultServiceConfiguration = configuration
-        } else {
-            self.useDefaultVoice = true
+    /**
+     `regionType` specifies what AWS region to use for Polly.
+     */
+    public var regionType: AWSRegionType = .USEast1
+    
+    
+    /**
+     `identityPoolId` is a required value for using AWS Polly voice instead of iOS's built in AVSpeechSynthesizer.
+     You can get a token here: http://docs.aws.amazon.com/mobile/sdkforios/developerguide/cognito-auth-aws-identity-for-ios.html
+     */
+    public var identityPoolId: String? {
+        didSet {
+            if let poolId = identityPoolId {
+                let credentialsProvider = AWSCognitoCredentialsProvider(regionType:regionType, identityPoolId: poolId)
+                let configuration = AWSServiceConfiguration(region:regionType, credentialsProvider:credentialsProvider)
+                AWSServiceManager.default().defaultServiceConfiguration = configuration
```

Comment:
Nvm, it looks like this is being inited twice. `useDefaultVoice` code looks good